psa-attestation/Cargo.toml

```diff
@@ -18,8 +18,11 @@ nitro = []
 
 [dependencies]
 libc = "0.2.69"
+# We are not really using psa-crypto but we are using the C libraries
+# libmbedcrypto.a and libshim.a that psa-crypto indirectly uses.
+psa-crypto = { path = "../third-party/rust-psa-crypto/psa-crypto" }
```

Comment:
Given the comment should the dependency rather be on `psa-crypto-sys`?

Reply:
It probably doesn't make any difference. I'll try changing it.